src/pykeen/evaluation/classification_evaluator.py

```diff
@@ -115,10 +129,10 @@ def process_head_scores_(
         if dense_positive_mask is None:
             raise KeyError("Sklearn evaluators need the positive mask!")
 
-        self._process_scores(keys=hrt_batch[:, 1:], scores=scores, positive_mask=dense_positive_mask, head_side=True)
+        self._process_scores(keys=hrt_batch[:, 1:], scores=scores, positive_mask=dense_positive_mask, side=LABEL_HEAD)
```

Comment:
since both, the rank-based and classification based evaluator use this pattern of delegating to a all-side helper method, we could consider making this the new API, and dropping the `process_head_scores_` etc. This should also simplify the code in the evaluation loop.